packages/components/src/query-controls/test/terms.ts

```diff
@@ -6,60 +6,76 @@ import { buildTermsTree } from '../terms';
 describe( 'buildTermsTree()', () => {
 	it( 'Should return same array as input with null parent and empty children added if parent is never specified.', () => {
 		const input = Object.freeze( [
-			{ id: 2232, dummy: true },
-			{ id: 2245, dummy: true },
+			{ id: 2232, name: 'foo', dummy: true },
+			{ id: 2245, name: 'baz', dummy: true },
```

Comment:
I added `name` here, as the terms we pass to `buildTermsTree()` should have names.